.gitignore

```diff
@@ -1,6 +1,318 @@
+# Created by https://www.toptal.com/developers/gitignore/api/vim,macos,webstorm,node
+# Edit at https://www.toptal.com/developers/gitignore?templates=vim,macos,webstorm,node
```

Comment:
Probably better to let toptal manage the yarn files.  Add ".yarn" and regenerate.